pkg/docker/docker.go

```diff
@@ -618,12 +591,12 @@ func updateImageWithInspect(image *api.Image, inspect *dockertypes.ImageInspect)
 
 // RemoveContainer removes a container and its associated volumes.
 func (d *stiDocker) RemoveContainer(id string) error {
+	ctx := context.Background() // no timeout
 	opts := dockertypes.ContainerRemoveOptions{
 		RemoveVolumes: true,
 		Force:         true,
 	}
-	return d.kubeDockerClient.RemoveContainer(id, opts)
-
+	return d.client.ContainerRemove(ctx, id, opts)
```

Comment:
similarly, lose `ctx` and `opts` in favor of passing what is assigned to them directly into the method call